README.md

```diff
@@ -1,6 +1,7 @@
-# WireGuard Tools
+# WireGuard Key Exchange
 
-WireGuard Tools consists of both a frontend and a backend to dynamically supply WireGuard keys to a FreiFunk Gateway offering BATMAN over VXLAN over WireGuard.
+WireGuard Key Exchange is a tool consisting of two parts: a frontend (broker) and a backend (daemon). The frontend (broker) is where the client can push (register) its key before connecting. The backend (daemon) is injecting those keys into a WireGuard instance.
```

Comment:
I don't feel daemon to reflects properly what it does.

Reply:
I agree here but client seems also wrong as it programs the backend, sets routes etc..

Reply:
Worker?

Reply:
worker sounds good.